integration/src/test/java/org/kie/hacep/KafkaUtilTest.java

```diff
@@ -162,42 +168,31 @@ public void close() {
 
     private Properties getConsumerConfig() {
         Properties consumerProps = new Properties();
-        consumerProps.setProperty("bootstrap.servers",
-                                  BROKER_HOST + ":" + BROKER_PORT);
-        consumerProps.setProperty("group.id",
-                                  "group0");
-        consumerProps.setProperty("client.id",
-                                  "consumer0");
-        consumerProps.put("auto.offset.reset",
-                          "earliest");
-        consumerProps.setProperty("key.deserializer",
-                                  "org.apache.kafka.common.serialization.StringDeserializer");
+        consumerProps.setProperty("bootstrap.servers", BROKER_HOST + ":" + BROKER_PORT);
+        consumerProps.setProperty("group.id", "group0");
+        consumerProps.setProperty("client.id", "consumer0");
+        consumerProps.put("auto.offset.reset", "earliest");
+        consumerProps.setProperty("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
         return consumerProps;
     }
 
     private Properties getProducerConfig() {
         Properties producerProps = new Properties();
-        producerProps.setProperty("key.serializer",
-                                  "org.apache.kafka.common.serialization.StringSerializer");
-        producerProps.setProperty("bootstrap.servers",
-                                  BROKER_HOST + ":" + BROKER_PORT);
+        producerProps.setProperty("key.serializer", "org.apache.kafka.common.serialization.StringSerializer");
+        producerProps.setProperty("bootstrap.servers", BROKER_HOST + ":" + BROKER_PORT);
         return producerProps;
     }
 
     public void createTopics(String... topics) {
         try {
             if (serverUp) {
-                for (String topic : topics) {
-
-                    if (!AdminUtils.topicExists(zkUtils, topic)) {
 
+                for (String topic : topics) {
+                    if(!adminClient.listTopics().listings().get().contains(topic)){
```

Comment:
This is not correct, `get()` returns a `Collection<TopicListing>` so you need to wrap `topic` in a `TopicListing` object